### PR TITLE
arr support in registerQueue

### LINKED ISSUE #5: support for array in registerQueue
If an array is passed, the same callback should be registered to all given queues

### PR DESCRIPTION
**Pull request by dlukanin**
Closes #5

### CODE REVIEW

**Comment by angrykoala** [comment]
Hi @dlukanin thanks for your contribution. Could you please commit your PR into dev branch (it is our way to bundle all changes before making the next release) thanks! :smile:

File: src/service.ts [hunk 1]
```diff
@@ -219,4 +217,15 @@ export default class RabbitMQService {
       this.assertedQueues.add(queueName)
     }
   }
+
+  private _registerQueue(queueName: string, cb: (r: any) => Promise<void>): void {
```
Comment (angrykoala) [inline src/service.ts]: Would it make sense to change the name to `registerSingleQueue` so the difference with `registerQueue` is more explicit?

**Comment by angrykoala** [comment]
thanks @dlukanin it's looking good, there are just a couple of minor things for you to review. Nice one on adding new test cases

**Comment by dlukanin** [comment]
Hey. Thx, I'll fix it today

**Pull request merged**